Automattic/newspack-popups
# Add an alignment/placement setting

### All Submissions:






### Changes proposed in this Pull Request:

Add an option to the pop-up in order to align it:
* Center
* Bottom
* Top
* Left
* Right

![Screenshot 2019-10-31 at 11 54 32](https://user-images.githubusercontent.com/177929/67944845-630e5400-fbd5-11e9-8837-121bb8d33d97.png)
![1 center](https://user-images.githubusercontent.com/177929/67945131-16774880-fbd6-11e9-931f-866ee261435a.png)
![2 bottom](https://user-images.githubusercontent.com/177929/67945132-170fdf00-fbd6-11e9-9a4d-6685ac3a6536.png)
![3 top](https://user-images.githubusercontent.com/177929/67945134-170fdf00-fbd6-11e9-9506-b9f54648bb06.png)
![4 left](https://user-images.githubusercontent.com/177929/67945135-170fdf00-fbd6-11e9-9913-71bb87778abe.png)
![5 right](https://user-images.githubusercontent.com/177929/67945136-17a87580-fbd6-11e9-8bdb-aae9e71a3061.png)



Closes #8.

### How to test the changes in this Pull Request:

1. Switch to this branch and run `npm run build`
2. Edit a Pop-up and open the popup settings 
3. Play with the different alignments and check the front-end.
4. Resize your screen (height and width). Can you still access the content of the pop-up.

### Other information:
